src/api/app/jobs/send_event_emails.rb

```diff
@@ -5,18 +5,12 @@ class SendEventEmails
   attr_accessor :event
 
   def perform
-    Event::Base.where(mails_sent: false).order(:created_at).limit(1000).lock(true).each do |event|
-      event.mails_sent = true
-      begin
-        event.save!
-      rescue ActiveRecord::StaleObjectError
-        # if someone else saved it too, better don't continue - we're not alone
-        return false
-      end
+    Event::Base.where(mails_sent: false).order(:created_at).limit(1000).each do |event|
```

Comment:
The `.order(:created_at)` here is ASC by default so the oldest already come first. But I will add `:asc` to the arguments to make it more clear.

https://apidock.com/rails/ActiveRecord/QueryMethods/order